tests/impact/client/utils.py

```diff
@@ -0,0 +1,105 @@
+from modelon.impact.client.entities import (
+    CustomFunction,
+    Workspace,
+    Model,
+    ModelExecutable,
+    Experiment,
+    Case,
+    ExternalResult,
+)
+from unittest.mock import MagicMock
+
+
+def get_workspace_entity(
```

Comment:
Sorry for the bad name suggestion 😄. The name prefix for these function should be `create_` not `get_` (still not 100% because you can mix it up with the actual create methods but should be better).